modules/layers/src/polygon-layer/polygon-layer.js

```diff
@@ -100,6 +100,10 @@ export default class PolygonLayer extends CompositeLayer {
     });
   }
 
+  unwrapObject(object) {
```

Comment:
For some reason this one in particular looks rather hacky...

Reply:
Yes it is. I'll clean up in a follow up PR.